routes/index.js

```diff
@@ -21,13 +21,59 @@ router.get("/myhabits", (req, res) => {
     res.redirect("/myhabits.html");
 });
 
-router.use((req, res, next) => {
-    res.status(404).sendFile(join(__dirname, "..", "public", "404.html"));
-});
+// router.use((req, res, next) => {
+//     res.status(404).sendFile(join(__dirname, "..", "public", "404.html"));
+// });
 
 /* ------Katerina end----- */
 
+/* ------Anshul Start ----- */
+router.post("/createHabit", fileUpload(), async (req, res) => {
+    console.log(req.files);
+    if (req.files) {
+        let file;
 
+        file = req.files.img;
+        let uploadPath = "./public/assets/images/" + file.name;
+        console.log(uploadPath);
+        try {
+            // Use the mv() method to place the file somewhere on your server
+            await file.mv(uploadPath, function (err) {
+                if (err) return res.status(500).send(err);
 
+                // res.send("Received!!!");
+            });
+            let { habitName, goalPerDay, startDate, numberOfDays, picture } =
+                req.body;
+            picture = uploadPath;
+            mongo.createHabits(
+                habitName,
+                goalPerDay,
+                startDate,
+                numberOfDays,
+                picture
+            );
+            res.send("Usign custom images");
+        } catch (e) {
+            console.log("Error", e);
+            res.status(400).send({ err: e });
+        }
+    } else {
+        // check which image has been selected and assign a value
+        // if its beach set the path in it as the
+        const { habitName, goalPerDay, startDate, numberOfDays, picture } =
+            req.body;
+        mongo.createHabits(
+            habitName,
+            goalPerDay,
+            startDate,
+            numberOfDays,
+            picture
+        );
+        console.log(req.body.picture);
+        res.send("Usign default images");
```

Comment:
after creating new habit, the user doesn't redirect to my habits page
```suggestion
        res.redirect("/myhabits.html");
```